app/src/main/res/layout-land/recently_played_fragment.xml

```diff
@@ -1,58 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
-
-  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Did we ever need landscape-specific specialization for this layout or did you just noticed it wasn't needed?

Reply:
Actually we do need other xml files too but if this approach gets approved the xml files will change a lot and therefore I thought to remove them directly for now, finish the implementation, then again add those files back with all the changes. That way i won't have to make changes to various versions of xml files again and again.